### PR TITLE
removing site and adding latlong to rucio did finder

### PR DESCRIPTION
**Pull request by ivukotic**
this addresses issues: 
https://github.com/ssl-hep/ServiceX/issues/305
and
https://github.com/ssl-hep/ServiceX_DID_Finder_Rucio/issues/35

### CODE REVIEW

File: servicex/templates/did-finder-rucio/deployment.yaml [hunk 1]
```diff
@@ -35,8 +35,10 @@ spec:
           - name: RMQ_URI
             value: amqp://user:{{ .Values.rabbitmq.auth.password }}@{{ .Release.Name }}-rabbitmq:5672
     {{- end }}
-          - name: SITE
-            value: {{ if .Values.didFinder.rucio.site }} --site {{ .Values.didFinder.rucio.site }} {{ end }}
+          - name: RUCIO_LATITUDE
```
Comment (sthapa) [inline servicex/templates/did-finder-rucio/deployment.yaml:38]: Is the lat and long being passed in using the --site argument?  If so, how does the code differentiate between the two?

**Review by sthapa — COMMENTED** [review]
I'm not seeing any code changes, did that accidentally get left out?

File: servicex/templates/did-finder-rucio/deployment.yaml (hunk 1 above)
Comment (ivukotic) [reply servicex/templates/did-finder-rucio/deployment.yaml:38]: this was messed up when redoing things. There is no argument. Rucio will simply take it from environment.

**Review by BenGalewsky — APPROVED** [review]
Looks good - I look forward to seeing how this plays out with the new Rucio release

**Pull request merged**